webapp/graphite/metrics_filter.py

```diff
@@ -0,0 +1,37 @@
+import re
+
+from django.conf import settings
+from graphite.metric_filter.models import UserMetricFilter
+from graphite.metric_filter.models import GroupMetricFilter
+
+
+def nodeFilter(user, nodePath):
+  if settings.ALLOW_METRICFILTER_BY_USER:
+    if user.id is not None:
+      filters = UserMetricFilter.getFilters(user.id)
+      if filters is not None:
+        for filter in filters.all():
+          matching = match(nodePath, filter)
+          if matching is not None:
+            return matching
+    for filters in GroupMetricFilter.getFiltersByUser(user):
+      if filters is not None:
+        for filter in filters.all():
```

Comment:
Same as above, please don't use variable names that override stdlib builtin functions. I'm almost certain a linter such as flake8 would pick this up as a no-no.